docs/firebase_layers.js

```diff
@@ -50,22 +50,22 @@ function createStyleFunction(colorFunctionProperties) {
  * base color to white.
  *
  * @param {String} field property whose value is used to determine color
- * @param {number} opacity
  * @param {number} minVal minVal of {@param field}
  * @param {number} maxVal maxVal of {@param field}
  * @param {String} color base color
  * @return {Function}
  */
-function createContinuousFunction(field, opacity, minVal, maxVal, color) {
+function createContinuousFunction(field, minVal, maxVal, color) {
   const colorRgb = colorMap.get(color);
   const white = colorMap.get('white');
   return (feature) => {
     const value = feature['properties'][field];
     const rgba = [];
     for (let i = 0; i < 3; i++) {
+      // https://www.alanzucconi.com/2016/01/06/colour-interpolation/
```

Comment:
Sure. Note that the only difference between this formula and the one we had before was the scaling. The old one just had a bug where we weren't scaling down properly (dividing by 2 instead of the range).